clients/html/src/app/config/client_configuration.ts

```diff
@@ -91,7 +91,7 @@ export const CLIENT_PREFERENCES: ClientPreferences = {
   tiered_package_types: [PackageTypes.SINGLE_PRODUCT],
   relationship_quote_calculator: RelationshipCoverageCostCalculatorService,
   tiered_quote_calculator: TieredCoverageCostCalculatorService,
-  default_state: 'MA',
+  default_state: 'ME',
```

Comment:
remove this `default_state` completely, seems like we're not using it anywhere.

Reply:
removed default_state